api/src/Entity/Camp.php

```diff
@@ -128,8 +150,8 @@ class Camp extends BaseEntity implements BelongsToCampInterface {
     #[InputFilter\CleanHTML]
     #[Assert\NotBlank]
     #[Assert\Length(max: 32)]
-    #[ApiProperty(example: 'Abteilungs-Sommerlager 2022')]
-    #[Groups(['Default', 'camp:update'])]
+    #[ApiProperty(writable: true, example: 'Abteilungs-Sommerlager 2022')]
```

Comment:
```suggestion
    #[ApiProperty(example: 'Abteilungs-Sommerlager 2022')]
```
`writable: true` ist der Default